Document icons for collapsible sections in services (#2291)
home-assistant/developers.home-assistant: docs/dev_101_services.md

The icon to use for each service action can be defined in the `icons.json` translation file in the integration folder, under the `services` key. The key should be the service action name, and the value should be the icon to use.

In addition, icons can optionally be specified for collapsible sections.

> 

Reviewer: I'd move this next to the example json showing this.

The following example, shows how to provide icons for the `turn_on` and `turn_off` service actions of an integration:

```json
{
  "services": {
    "turn_on": {"service": "mdi:lightbulb-on"},
    "turn_off": {"service": "mdi:lightbulb-off"}
  }
}
```

> The following example, shows how to provide icons for a service with sections:

Reviewer: ```suggestion
The following example shows how to provide an icon to the `advanced_options` section:
```
How about something like this?
